package.json

```diff
@@ -8,41 +8,6 @@
     ]
   },
   "scripts": {
-    "build:hello": "yarn workspace @hono/hello build",
-    "build:zod-validator": "yarn workspace @hono/zod-validator build",
-    "build:class-validator": "yarn workspace @hono/class-validator build",
-    "build:arktype-validator": "yarn workspace @hono/arktype-validator build",
-    "build:qwik-city": "yarn workspace @hono/qwik-city build",
-    "build:graphql-server": "yarn workspace @hono/graphql-server build",
-    "build:sentry": "yarn workspace @hono/sentry build",
-    "build:firebase-auth": "yarn workspace @hono/firebase-auth build",
-    "build:trpc-server": "yarn workspace @hono/trpc-server build",
-    "build:clerk-auth": "yarn workspace @hono/clerk-auth build",
-    "build:typebox-validator": "yarn workspace @hono/typebox-validator build",
-    "build:medley-router": "yarn workspace @hono/medley-router build",
-    "build:valibot-validator": "yarn workspace @hono/valibot-validator build",
-    "build:zod-openapi": "yarn workspace @hono/zod-openapi install && yarn workspace @hono/zod-openapi build",
-    "build:typia-validator": "yarn workspace @hono/typia-validator build",
-    "build:swagger-ui": "yarn workspace @hono/swagger-ui build",
-    "build:swagger-editor": "yarn workspace @hono/swagger-editor build",
-    "build:esbuild-transpiler": "yarn workspace @hono/esbuild-transpiler build",
-    "build:event-emitter": "yarn workspace @hono/event-emitter build",
-    "build:oauth-providers": "yarn workspace @hono/oauth-providers build",
-    "build:react-renderer": "yarn workspace @hono/react-renderer build",
-    "build:auth-js": "yarn workspace @hono/auth-js build",
-    "build:bun-transpiler": "yarn workspace @hono/bun-transpiler build",
-    "build:prometheus": "yarn workspace @hono/prometheus build",
-    "build:oidc-auth": "yarn workspace @hono/oidc-auth build",
-    "build:node-ws": "yarn workspace @hono/node-ws build",
-    "build:react-compat": "yarn workspace @hono/react-compat build",
-    "build:effect-validator": "yarn workspace @hono/effect-validator build",
-    "build:conform-validator": "yarn workspace @hono/conform-validator build",
-    "build:casbin": "yarn workspace @hono/casbin build",
-    "build:ajv-validator": "yarn workspace @hono/ajv-validator build",
-    "build:tsyringe": "yarn workspace @hono/tsyringe build",
-    "build:cloudflare-access": "yarn workspace @hono/cloudflare-access build",
-    "build:standard-validator": "yarn workspace @hono/standard-validator build",
-    "build:otel": "yarn workspace @hono/otel build",
```

Comment:
I'm 90% sure these aren't used anywhere 👀

Reply:
You are correct. It's okay to remove them!